src/ILogger/ILogger.csproj

```diff
@@ -8,6 +8,7 @@
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
     <RunCodeAnalysis>false</RunCodeAnalysis>
+    <DebugType>Full</DebugType>
```

Comment:
What’s going on here?

Reply:
Removed that line. Was trying to find the paths not tested by unit tests. I forgot that newer SDK had the fix for unit tests.